.semaphore/goreleaser.yml

```diff
@@ -19,15 +17,9 @@ blocks:
           - 'mkdir -vp "${SEMAPHORE_GIT_DIR}" "${GOPATH}/bin"'
           - export SEMAPHORE_CACHE_DIR=/home/semaphore
           - checkout
-          - make install-vault
-          - . vault-bin/vault-setup
-          - . vault-sem-get-secret ssh_id_rsa
-          - . vault-sem-get-secret netrc
-          - . vault-sem-get-secret ssh_config
-          - . vault-sem-get-secret gitconfig
+          - vault-setup
```

Comment:
the promotion for semaphore.yml doesn't have any conditions. We either need to add conditions so that it doesn't run on public forked PRs or remove Vault completely.

Reply:
I expect the gorelease won't be run on public CI but will be a release step on internal CI only.